application/src/View/Helper/AssetUrl.php

```diff
@@ -83,7 +83,7 @@ public function __invoke($file, $module = null, $override = false, $versioned =
             && ($module === 'Omeka' || array_key_exists($module, $this->activeModules))
         ) {
             $themeId = $this->currentTheme->getId();
-            $filepath = sprintf(self::THEME_ASSETS_PATH, OMEKA_PATH, $themeId, $file, '');
+            $filepath = sprintf(substr(self::THEME_ASSETS_PATH, 0, -5), OMEKA_PATH, $themeId, $file);
```

Comment:
This isn't actually broken in its current form, is it? And doesn't the `-5` substring chop off too much of the template here, losing the actual filename part?

I don't like the clunkiness of the current sprintf here either, though... it'd probably be better to drop the extra `%s` from all the templates and just factor out the addition of the version GET parameter to another function or something like that.